liiatools_pipeline/sensors/sufficiency_sensor.py

```diff
@@ -7,17 +7,16 @@
 )
 from liiatools_pipeline.jobs.ssda903_la import ssda903_clean
 from liiatools_pipeline.jobs.ssda903_org import ssda903_sufficiency
+from dagster import RunRequest, RunsFilter, DagsterRunStatus, sensor
+from liiatools_pipeline.jobs.common_la import clean
+from liiatools_pipeline.jobs.common_org import ssda903_sufficiency
 
 
-@sensor(
-    job=ssda903_sufficiency,
-    description="Completion of the 903 cleanfile job before running the sufficiency process for 903 Files",
-    default_status=DefaultSensorStatus.RUNNING,
-)
+@sensor(job=ssda903_sufficiency)
 def sufficiency_sensor(context):
     run_records = context.instance.get_run_records(
         filters=RunsFilter(
-            job_name=ssda903_clean.name,
+            job_name=clean.name,
```

Comment:
I think we'll need a way to determine if a 903 clean has been run, not sure how to do this just yet, and it will go in the sensors PRs but one to think about